docs/taskruns.md

```diff
@@ -766,11 +768,35 @@ or fails from an error.
 You can execute the `Task` in your `TaskRun` with a specific set of credentials by
 specifying a `ServiceAccount` object name in the `serviceAccountName` field in your `TaskRun`
 definition. If you do not explicitly specify this, the `TaskRun` executes with the credentials
-specified in the `configmap-defaults` `ConfigMap`. If this default is not specified, `TaskRuns`
+specified in the `configmap-defaults` `ConfigMap`. If this defaulPipelineRunt is not specified, `TaskRuns`
 will execute with the [`default` service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server)
 set for the target [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
 
 For more information, see [`ServiceAccount`](auth.md).
+## `TaskRun` status
+The `status` field defines the observed state of `TaskRun`
+### The `status` field
+The `TaskRun`'s  `status` field can contain the following fields:
+
+- Required:
+  - `status` - The most relevant information about the TaskRun's state. This field includes:
+    - `status.conditions`, which contains the latest observations of the `TaskRun`'s state. [See here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties) for information on typical status properties. 
+  <!-- - `podName` - Name of the pod containing the containers responsible for executing this `task`'s `step`s. -->
+  - `startTime` - The time at which the `TaskRun` began executing, conforms to [RFC3339](https://tools.ietf.org/html/rfc3339) format.
+  - `completionTime` - The time at which the `TaskRun` finished executing, conforms to [RFC3339](https://tools.ietf.org/html/rfc3339) format.
+  - [`taskSpec`](tasks.md#configuring-a-task) - `TaskSpec` defines the desired state of the `Task` executed via the `TaskRun`.
+
+- Optional:
+  - `taskResults`- Describes the results of this `task`.
+
+  - `Provenance` - Provenance contains metadata about resources used in the `TaskRun` such as the source from where a remote `task` definition was fetched. It carries minimum amount of metadata in `TaskRun` `status` so that `Tekton Chains` can utilize it for provenance, its two subfields are:
```

Comment:
`provenance` (lower case)

Reply:
this comment made everything clear.
I couldn't think through this naming used, now I have some idea. thanks